qn-7.js

```diff
@@ -0,0 +1,119 @@
+// number one
+const products = [
+  { name: "Product 1", count: 10, price: 100 },
+  { name: "Product 2", count: 5, price: 50 },
+  { name: "Product 3", count: 20, price: 200 },
+  { name: "Product 5", count: 0, price: 50 },
+
+];
+
+function getTotalItemCount() {
+  let totalCount = 0;
+  products.forEach((product) => {
+    totalCount += product.count;
+  });
+  console.log(totalCount);
+}
+getTotalItemCount();
+
+//   number two
+function createNewItem(name, count, price) {
+  const newItem = { name, count, price };
+  products.forEach(item => {
+  if(item.name !== newItem.name){
```

Comment:
By using `id` you could ensure that the item is unique indeed in case that two products have the same name.